.eslintrc

```diff
@@ -15,11 +16,29 @@
   },
   "plugins": ["@typescript-eslint"],
   "rules": {
+    "eslint-comments/no-unused-disable": "error",
     "prefer-const": "error",
     "@typescript-eslint/no-unsafe-member-access": "off",
     "@typescript-eslint/no-unsafe-call": "off",
     "@typescript-eslint/no-unsafe-assignment": "off",
     "@typescript-eslint/no-unsafe-return": "off",
```

Comment:
Can we validate if these should still be off?

Reply:
@sgraczyk I went through these rules and enabled the rules that could be applied to the project ✅.

But there are issues with `no-unsafe-member-access` and `no-unsafe-assignment`. Our `design-system-icons` package does not generate proper types for icons, and because of this, TypeScript is unable to match the type from the `source` prop in the `<Icon>` component with generated icon components from the icons library. I've marked these two rules with TODO for this PR.

![image](https://github.com/livechat/design-system/assets/12149393/f5b77b83-e89e-4930-947b-39ac4c3f4e5c)